truss/templates/cache.Dockerfile.jinja

```diff
@@ -1,14 +1,17 @@
 FROM python:3.11-slim as cache_warmer
 
-RUN mkdir -p /app/hf_cache
+RUN mkdir -p /app/model_cache
 WORKDIR /app
 
 {% if hf_access_token %}
 ENV HUGGING_FACE_HUB_TOKEN {{hf_access_token}}
 {% endif %}
-{%- if credentials_exists %}
+{%- if gcs_credentials_exists %}
```

Comment:
we repeat these magic file paths in a lot of places, I wonder if we could just pass in credentials here, and can just do:

`COPY ./data/{{ credentials}} ...` 

instead of having branching logic? And then we can define these constants in one place